src/commands/translation/push.ts

```diff
@@ -85,6 +85,8 @@ export default class TranslationPush extends BaseCommand<
     spinner.start(`‣ Pushing`);
 
     for (const translation of translations) {
+      if (translation.namespace === "system") continue;
```

Comment:
I think it's better to pull out this "system" value as a constant rather than hardcoding it in a ad hoc fashion i.e. something like `const SYSTEM_NAMESPACE = "system"`. And probably define it in `lib/marshal/translation/helpers.ts` rather than in a command specific module.